README.md

```diff
@@ -281,6 +281,53 @@ NAME                                            READY   STATUS    RESTARTS   AGE
 vault-kubernetes-synchronizer-7d5f65895-2pf4j   1/1     Running   0          5m18s
 ```
 
+## Example - with failed synchronizer
+
+### Permission issue
+```
+$ k logs pod/vault-kubernetes-synchronizer-demo-vvzxr -c vault-kubernetes-synchronizer
+2019/08/20 14:00:28 Using annotation [ vault-secret ] to detect managed secrets
+2019/08/20 14:00:28 failed to prepare synchronization of secrets: Error making API request.
+
+URL: GET http://internal-Paxata-Dev-vault-ELB-1522627047.us-west-2.elb.amazonaws.com/v1/sys/mounts
```

Comment:
Could you please change this URL to `http://example.com/v1/....`?

Reply:
Sure I will change accordingly.